### PR TITLE
Redirect crash reports to fastly

### PR DESCRIPTION
**Pull request by keur**
* Crash reports now handled by a third-party (backtrace)
  * Set 301 manually, since hapi default is 302

### CODE REVIEW

File: src/controllers/crashes.js [hunk 1]
```diff
@@ -37,19 +40,13 @@ exports.setup = (runtime) => {
...
-        delete payload.guid
-        runtime.mongo.models.insertCrash(payload, 'braveCore', (err, results) => {
-          console.log(`crash recorded for version ${payload.ver}`)
-        })
+      description: "Proxy crash reports to Fastly endpoint",
+      handler: {
+        proxy: {
+          uri: process.env.CRASH_PROXY,
+          passThrough: true,
+          timeout: 10000
```
Comment (bridiver) [inline src/controllers/crashes.js]: what is this timeout value in? Seconds or milliseconds?

Comment (bridiver) [reply src/controllers/crashes.js]: if it's milliseconds it might be too short for people with slow connections to complete the minidump upload

Comment (bridiver) [reply src/controllers/crashes.js]: and if it's seconds then it's probably way too long :)

Comment (keur) [reply src/controllers/crashes.js]: milliseconds. do we want to bump it up?

Comment (keur) [reply src/controllers/crashes.js]: how long do we want it set for? @bridiver

**Pull request merged**